react/components/ProductSpecifications/index.js

```diff
@@ -0,0 +1,126 @@
+import React, { Component, Fragment } from 'react'
+import PropTypes from 'prop-types'
+import { FormattedMessage, injectIntl, intlShape } from 'react-intl'
+import HtmlParser from 'react-html-parser'
+import GradientCollapse from '../GradientCollapse/index'
+
+import { Tabs, Tab } from 'vtex.styleguide'
+
+import styles from './styles.css'
+
+/**
+ * Product Specification Component.
+ * Render the technical specifications of a product. Can be displayed in two views: Table view or Tabs view.
+ */
+class ProductSpecifications extends Component {
```

Comment:
_Just a tip:_ You can use [hooks](https://reactjs.org/docs/hooks-overview.html) instead of classes, but no need for that now